bluemira/base/builder.py

```diff
@@ -111,6 +113,16 @@ def __init__(
         self.name = self.build_config.get(
             "name", _remove_suffix(self.__class__.__name__, "Builder")
         )
+        self._build = self.build
+        self.build = self._build_wrapper
+
+    def _build_wrapper(self) -> Component:
+        """Build the component."""
+        bluemira_print(f"Building {self.name}")
+        t1 = time.time()
+        out = self._build()
+        bluemira_debug(f"Built in {time.time() - t1} s")
```

Comment:
Could we have some pretty time formatting? i.e. `:.2f` or some `datetime.now().strpf()` or whatever?